internal/repository/sync_test.go

```diff
@@ -196,7 +183,7 @@ func TestClone(t *testing.T) {
 		defer os.Chdir(currentDir) //nolint:errcheck
 
 		dirName := "myRepo"
-		if err := os.Mkdir(dirName, 0755); err != nil {
+		if err := os.Mkdir(dirName, 0o755); err != nil {
```

Comment:
unrelated change? `0755` is already octal no? Looks like go supports `0`, `0o`, and `0O`: https://go.dev/ref/spec#Integer_literals

Reply:
This was a typo, did not mean to change.